docs/configuration/snap.md

```diff
@@ -3,7 +3,7 @@ The top-level [snap](configuration.md#Configuration-snap) key contains set of op
 <!-- do not edit. start of generated block -->
 <ul>
 <li>
-<p><code id="SnapOptions-base">base</code> String | “undefined” - A snap of type base to be used as the execution environment for this snap. Examples: <code>core</code>, <code>core18</code>, <code>core20</code>. Defaults to <code>core18</code></p>
+<p><code id="SnapOptions-base">base</code> String | “undefined” - A snap of type base to be used as the execution environment for this snap. Examples: <code>core</code>, <code>core18</code>, <code>core20</code>, <code>core22</code>. Defaults to <code>core18</code></p>
```

Comment:
Similar to https://github.com/electron-userland/electron-builder/pull/7814#discussion_r1359493735

You'll need to add it to the file here: https://github.com/electron-userland/electron-builder/blob/2ca621289e025a6d0856c7c588aaac40d267cb5f/packages/app-builder-lib/src/options/SnapOptions.ts#L6
Then run `pnpm generate-all`